docs/antora.yml

```diff
@@ -0,0 +1,3 @@
+name: epub3
```

Comment:
I think it should be `epub3-converter` if we want to stay consistent with `reveal.js-converter`

Reply:
You're right, fixed.